keras/src/layers/preprocessing/stft_spectrogram.py

```diff
@@ -0,0 +1,410 @@
+import math
+import warnings
+
+from keras.src import backend
+from keras.src import initializers
+from keras.src import layers
+from keras.src import ops
+from keras.src.api_export import keras_export
+from keras.src.utils.module_utils import scipy
+
+
+class STFTInitializer(initializers.Initializer):
+    def __init__(self, side, window, scaling, periodic=None):
```

Comment:
Is `periodic` supposed to be boolean? Why is it `None` rather than `False`?

Reply:
This was a mistake, made it to `False` by default.